### PR TITLE
fix: added commissions for orga users

### PR DESCRIPTION
**Pull request by TeddyRoncin**
# Add a support to store commissions for each orga

It also allowed for the implementation of the trombi

## Changes

* added field orgaRoles in table users
* added table OrgaRole
* added table Commission
* added route GET /users/orgas
* removed user type orga as it was unused
* added permission orga
* added field orgaRoles createFakeUser to specify roles to the created user
* added field orgaRoles in PATCH /admin/users/:userId
* added route GET /commissions
* added field orgaRoles in detailed returned users

## Breaking changes

* "orga" user type does not exist anymore

### CODE REVIEW

File: src/controllers/users/openapi.yml [hunk 1]
```diff
@@ -244,3 +244,48 @@
...
+                        items:
+                          type: object
+                          properties:
+                            id:
+                              type: string
+                            firstname:
+                              type: string
+                            lastname:
+                              type: string
+                      member:
```
Comment (DevNono) [inline src/controllers/users/openapi.yml:281]: Mauvaise indentation

**Review by DevNono — CHANGES_REQUESTED** [review]
Autoriser le fetchOrgas uniquement lorsque le settings trombi est ON

File: src/controllers/users/openapi.yml (hunk 1 above)
Comment (TeddyRoncin) [reply src/controllers/users/openapi.yml:285]: humm, je crois pas

**Review by DevNono — APPROVED** [review]

File: src/controllers/users/openapi.yml [hunk 2]
```diff
@@ -247,7 +247,7 @@
 
 /users/orgas:
   get:
-    summary: Retourne la liste des organisateurs, avec leurs commissions et rôle dans chaque commission
+    summary: Retourne la liste des organisateurs, avec leurs commissions et rôle dans chaque commissions
```
Comment (DevNono) [inline src/controllers/users/openapi.yml:250]: Euh t'es sûr ?

**Review by DevNono — CHANGES_REQUESTED** [review]

**Pull request merged**